src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Features/GetAzureProviderFeatureCmdlet.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
     /// Gets the preview features of a certain azure resource provider.
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmProviderFeature", DefaultParameterSetName = GetAzureProviderFeatureCmdlet.ListAvailableParameterSet)]
-    [OutputType(typeof(List<PSProviderFeature>))]
+    [OutputType(typeof(PSProviderFeature))]
```

Comment:
@twitchax same comment about the output in the markdown for cmdlets not being updated (in Resources)

- [`Get-AzureRmProviderFeature.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmRoleDefinition.md#outputs)
- [`Get-AzureRmLocation.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmLocation.md#outputs)
- [`Register-AzureRmProviderFeature.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Register-AzureRmProviderFeature.md#outputs)
- [`Unregister-AzureRmResourceProvider.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Unregister-AzureRmResourceProvider.md#outputs)
- [`Get-AzureRmResourceGroupDeployment.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmResourceGroupDeployment.md#outputs)
- [`Test-AzureRmResourceGroupDeployment.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Test-AzureRmResourceGroupDeployment.md#outputs)
- [`Get-AzureRmRoleAssignment.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmRoleAssignment.md#outputs)
- [`Remove-AzureRmRoleAssignment.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Remove-AzureRmRoleAssignment.md#outputs)
- [`Get-AzureRmRoleDefinition.md`](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmRoleDefinition.md#outputs)